scripts/al2022/install-neuron-packages.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```

Comment:
Can we just reuse the existing install-additional-packages script for this?

Reply:
I am getting errors like
```
==> amazon-ebs.al2022neu: + sudo yum localinstall -y '/tmp/additional-packages/*.x86_64.rpm'
    amazon-ebs.al2022neu: Last metadata expiration check: 0:00:31 ago on Thu 26 May 2022 09:19:27 PM UTC.
==> amazon-ebs.al2022neu: Can not load RPM file: /tmp/additional-packages/*.x86_64.rpm.
==> amazon-ebs.al2022neu: Could not open: /tmp/additional-packages/*.x86_64.rpm
```
When I have noarch rpms under this directory. I am not really sure why it happened though...

Reply:
yep I will take a look